kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/write/row/GCSToBQWriter.java

```diff
@@ -100,11 +98,11 @@ public GCSToBQWriter(Storage storage,
    * @param blobName the name of the GCS blob to write.
    * @throws InterruptedException if interrupted.
    */
-  public void writeRows(List<RowToInsert> rows,
+  // writeRows -> needs to be changed
```

Comment:
?

Reply:
oops, left this here by mistake. Will remove it.